Library/Services/Implementation/NfieldSurveySampleService.cs

```diff
@@ -55,6 +56,18 @@ public Task<SampleUploadStatus> PostAsync(string surveyId, string sample)
                 .ContinueWith(stringResult => JsonConvert.DeserializeObject<SampleUploadStatus>(stringResult.Result))
                 .FlattenExceptions();
         }
+        
+        public Task<SampleUploadStatus> PostJsonAsync<TContent>(string surveyId, TContent sample)
```

Comment:
You should add at least one [unit test](https://github.com/NIPOSoftware/Nfield-SDK/blob/master/Tests/Services/NfieldSurveySampleTests.cs) for this